src/main/java/org/opensearch/sdk/handlers/LocalNodeResponseHandler.java

```diff
@@ -46,4 +65,15 @@ public String executor() {
     public LocalNodeResponse read(StreamInput in) throws IOException {
         return new LocalNodeResponse(in);
     }
+
+    /**
+     * Invokes await on the LocalNodeResponseHandler count down latch
+     */
+    public void awaitResponse() throws InterruptedException {
+        inProgressLatch.await(ExtensionsOrchestrator.EXTENSION_REQUEST_WAIT_TIMEOUT, TimeUnit.SECONDS);
+    }
+
+    public DiscoveryNode getLocalNode() {
```

Comment:
Small test to verify this getter?

Reply:
Since with the given implementation it is allowed to be null I'm not sure how it can be tested.